core/src/main/java/org/bitcoinj/crypto/ECKey.java

```diff
@@ -1012,7 +1032,11 @@ public static ECKey recoverFromSignature(int recId, ECDSASignature sig, Sha256Ha
         return ECKey.fromPublicOnly(q, compressed);
     }
 
-    /** Decompress a compressed public key (x co-ord and low-bit of y-coord). */
+    /**
+     * Decompress a compressed public key (x co-ord and low-bit of y-coord).
+     * (if <code>yBit</code> is <code>true</code>, the odd y coordinte will be used,
+     * if it is <code>false</code> the even y coordinate will be used)
+     */
```

Comment:
Since this is a JavaDoc comment, you could use `@param` tags to document `yBit` and add `xBN` as well.

Reply:
Done: Added the parameters as `@param` tags, also changed the overall text (existing inconsistent wording, `x co-ord ` vs. `y-coord`).

Reply:
I think the parameter rename should have gone into a separate PR or commit. Reading your PR description, this PR is about comments, has no code changes.

Reply:
Oh, yes, you are totally right. I will remove it again from here.

Reply:
@schildbach Done ✅ 

Now this PR should only contain comment changes (again). 🙂

Reply:
Squashed the commits tohgether, now it's one single commit, without code changes.